trade-finance-manager-ui/server/controllers/case/amendments/validation/amendmentBanksDecisionChoice.validate.js

```diff
@@ -0,0 +1,30 @@
+const { validationErrorHandler } = require('../../../../helpers/validationErrorHandler.helper');
+
+/**
+ *
+ * @param {String} decision
+ * @returns {Object}
+ * Validation function to check if decision has been selected on amendment banks decision
+ * if decision blank, then produces error and returns
+ * if decision, then returns blank error object
+ */
+const amendmentBankDecisionChoiceValidation = (decision) => {
+  if (!decision) {
+    const amendmentBankDecisionValidationErrors = [];
+
+    amendmentBankDecisionValidationErrors.push({
+      errRef: 'banksDecisionChoiceValue',
```

Comment:
`banksDecision` instead of `anksDecisionChoiceValue`

Reply:
done